pyproject.toml

```diff
@@ -2,7 +2,19 @@
 name = "layered-config"
 version = "2.0.0"
 description = "A tool for managing layered config files!"
-authors = ["Ryan Casperson <casperson.ryan@gmail.com>"]
+authors = ["Doug Philips <dgou@mac.com>", "Ryan Casperson <casperson.ryan@gmail.com>"]
```

Comment:
Sorry I had looked at the other PR first. As mentioned there, I think just having you here is most accurate and good with me.

Reply:
WF.